### ISSUE TITLE
undefined method `logger' for WSDiscovery::Searcher:Class

### ISSUE THREAD

**Issue by hackeron** [pelco-automation/ws-discovery#4]
This is what I get when I try to require 'ws_discovery' - any suggestions?

```ruby
[1] pry(main)> require 'ws_discovery'
NoMethodError: undefined method `logger' for WSDiscovery::Searcher:Class
from /usr/local/lib/ruby/gems/2.2.0/gems/ws_discovery-0.0.3/lib/ws_discovery/searcher.rb:9:in `<class:Searcher>'
```

**Comment by turboladen** [comment]
Probably the result of the log_switch gem going 1.0.0 and ws-discovery's dependency isn't locking to less than that.

**Comment by tindron** [comment]
Fixed in 0.0.4 by switching to semantic_logger.
